lms/templates/ux/reference/teams-base.html

```diff
@@ -155,11 +155,218 @@ <h1 class="page-title">Solar Power</h1>
     <div class="page-content">
       <nav class="page-content-nav">
         <a href="" class="nav-item is-active">My Teams</a>
-        <a href="" class="nav-item">Browse All Teams</a>
+        <a href="" class="nav-item">Browse</a>
       </nav>
 
       <div class="page-content-main">
-        <p>Blah blah blah. Go Team!</p>
+
+        <div class="search-tools">
+          <span class="search-count">Viewing 12 of 24 topics</span>
+          <span class="search-sort">sorted by
+          <select name="search-sort">
+            <option>number of teams</option>
+            <option>alphabetically</option>
+            <option>latest activity</option>
+          </select>
+        </div>
+
+
+        <div class="card square-card topic-card has-pennant">
+          <div class="card-core-wrapper">
+            <div class="card-core">
+              <p class="card-type pennant">Topic</p>
+              <h3 class="card-title">Renewable Energy</h3>
+              <p class="card-description">Explore how changes in renewable energy production will change our day-to-day lives</p>
+            </div>
+          </div>
+          <div class="card-meta-wrapper has-actions">
+            <div class="card-meta-details">
+              <p class="meta-detail team-count">34 Teams</p>
+            </div>
+            <div class="card-actions">
+              <a class="action action-view" href=""><span class="sr">View Teams in the Renewable Energy Topic</span><i class="icon fa fa-arrow-right"></i></a>
+            </div>
+          </div>
+        </div>
+
+        <div class="card square-card topic-card has-pennant">
+          <div class="card-core-wrapper">
+            <div class="card-core">
+              <p class="card-type pennant">Topic</p>
+              <h3 class="card-title">Solar Power Cells</h3>
+              <p class="card-description">Discuss the current technology trajectory of solar power cells</p>
+            </div>
+          </div>
+          <div class="card-meta-wrapper has-actions">
+            <div class="card-meta-details">
+              <p class="meta-detail team-count">34 Teams</p>
+            </div>
+            <div class="card-actions">
+              <a class="action action-view" href=""><span class="sr">View Teams in the Solar Power Cells Topic</span><i class="icon fa fa-arrow-right"></i></a>
```

Comment:
I think @cptvitamin recommends adding an `` aria-hidden="true"` to icons that have descriptive text provided next to them.